pages/doc.js

```diff
@@ -8,27 +8,22 @@ import { RightPanel } from '../src/Documentation/RightPanel/RightPanel'
 import Page from '../src/Page'
 import SearchForm from '../src/SearchForm'
 import Page404 from '../src/Page404'
-import PerfectScrollbar from 'perfect-scrollbar'
```

Comment:
let's keep it for now. Scrolling behave differently if you remove it and `wheelPropagation: true` stuff. Also, would need to check different browsers.

Reply:
It's moved to the `/src/Documentation/SidebarMenu/SidebarMenu.js`, because it requires  `#sidebar-menu` that is created in that file.